zerver/tests/test_auth_backends.py

```diff
@@ -667,6 +668,9 @@ def setUp(self) -> None:
         from social_core.backends.utils import load_backends
         load_backends(settings.AUTHENTICATION_BACKENDS, force_load=True)
 
+    def logger_output(self, output_string: str, type: str) -> str:
+        return f'{type.upper()}:zulip.auth.{self.backend.name}:{output_string}'
+
```

Comment:
The `m.output` of all `assertLogs` is a list containing a string with a format like the following string.
` WARN:zulip.auth.saml:SAML got invalid email argument.`. Instead of repeatedly writing the long `WARN:zulip.auth.saml:`, having a function felt better.

Reply:
OK, makes sense.